server/testutils/factories/factoryHelpers.ts

```diff
@@ -8,6 +8,16 @@ export default class FactoryHelpers {
     return factory.buildList(quantity)
   }
 
+  static buildListWith<T>(factory: Factory<T>, properties: DeepPartial<T>, count: number): Array<T> {
```

Comment:
Array
 .from({ length })
 .map ...

Reply:
`Array.from({ length })` - looks like this creates an empty array?

Reply:
Could probably do `Array.from(Array(5)).map` but it's a bit hacky

Reply:
No, just set { length:5 }, unlike new Array(5) it's enumerable. Your approach is fine though,